installing/installing-preparing.adoc

```diff
@@ -0,0 +1,86 @@
+
+You can use the installer-provisioned infrastructure method to create appropriate machine instance in your data center for xref:../installing/installing_rhv/installing-rhv-default.adoc#installing-rhv-default[{rh-virtualization}], xref:../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[vSphere], and xref:../installing/installing_bare_metal_ipi/ipi-install-overview#ipi-install-overview[bare metal].
+
+If you want to reuse extensive cloud infrastructure, you can complete a _user provisioned infrastructure_ installation. With these installations, you manually deploy the machines that your cluster requires during the installation process. If you perform a user provisioned infrastructure installation on xref:../installing/installing_aws/installing-aws-user-infra.adoc#installing-aws-user-infra[AWS], xref:../installing/installing_azure/installing-azure-user-infra.adoc#installing-azure-user-infra[Azure], or xref:../installing/installing_gcp/installing-gcp-user-infra.adoc#installing-gcp-user-infra[GCP], you can use the provided templates to help you stand up all of the required components. You can also use the provider agnostic installation method to deploy a cluster into other clouds.
+
+You can also complete a user provisioned infrastructure installation on hardware in your data center. If you use xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#installing-openstack-installer-custom[{rh-openstack}], xref:../installing/installing_rhv/installing-rhv-user-infra.adoc#installing-rhv-user-infra[{rh-virtualization}], xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[IBM Z or LinuxONE], xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power[IBM Power], or xref:../installing/installing_vsphere/installing-vsphere.adoc#installing-vsphere[vSphere], use the specific installation instructions to deploy your cluster. If you use other supported hardware, follow the xref:../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[bare metal installation] procedure.
+
+[id="installing-preparing-security"]
+=== Do you need extra security for your cluster?
```

Comment:
I would order this section in somewhat increasing levels of security. When I read it the first time, I got to the first section where it talks about "limited access to the internet" and conflated that with a private cluster. Then I was confused as to why that required user-provisioned installation.
1. Proxy
2. Private
3. Disconnected/Restricted
4. Gov cloud
5. FIPS